<launch> 
 <param name="/use_sim_time" value="true"/>
  <!-- Map server -->
  <arg name="map_file" default="$(find arobota2)/map/mymap5.yaml"/>
  <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)">
    <param name="frame_id" value="/map" />
  </node>

  <!-- move_base of multiple robot-->

  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1_tf" />
    <param name="amcl/initial_pose_x" value="0.4" />
    <param name="amcl/initial_pose_y" value="0.4" />
    <include file="$(find arobota2)/launch/movebase_gazebo.launch"/>
  </group>
  <!-- BEGIN ROBOT 2-->
  <group ns="robot2">
    <param name="tf_prefix" value="robot2_tf" />
    <param name="amcl/initial_pose_x" value="-0.4" />
    <param name="amcl/initial_pose_y" value="0.4" />
    <include file="$(find arobota2)/launch/movebase_gazebo.launch"/>
  </group>
  <!-- BEGIN ROBOT 3-->
    <group ns="robot3">
    <param name="tf_prefix" value="robot3_tf" />
    <param name="amcl/initial_pose_x" value="0" />
    <param name="amcl/initial_pose_y" value="0" />
    <include file="$(find arobota2)/launch/movebase_gazebo.launch"/>
  </group>
  <!-- rviz -->
  <node name="rviz" pkg="rviz" type="rviz" required="true" args="-d $(find arobota2)/rviz/navigation.rviz"/>
</launch>